website/markdown/docs/stats/about.mdx

```diff
@@ -0,0 +1,26 @@
+---
+name: Stats
+exceprt: Learn about Tuist Stats
+---
+
+# About Stats
+
+Tuist sends some **anonymous** analytics events to track the usage of the tool.
+Having statistics about the usage helps contributors to take informed decision and better prioritize future work.
+For example, if a command is only used by 1% of the users, it might not get prioritized for enhancements in the future.
+
+The events that are tracked are listed on this documentation. The implementation is open source, mainly in the [TuistAnalytics](https://github.com/tuist/tuist/tree/main/Sources/TuistAnalytics) target.
+If you are thinking about adding another event, please remember that we would like to keep tracking minimal and unobtrusive.
+
+## Can I see the data?
```

Comment:
Instead of organizing this page as if it was a "FAQ" what about using the narrative.